src/browser/browser.ts

```diff
@@ -381,10 +383,18 @@ export class Browser {
           height: scrollResult.scrollHeight,
         });
       }
+
+      if (isPDF) {
+        return page.pdf({
+          ...getPDFOptionsFromURL(options.url),
+          path: options.filePath,
+        })
+      }
+
       return page.screenshot({ path: options.filePath, fullPage: options.fullPageImage, captureBeyondViewport: options.fullPageImage || false });
```

Comment:
should we wrap this in if(!isPDF ) ?

Reply:
not necessary given the `if(isPDF) ...` case returns, but perhaps an else clause would be cleaner (I don't have any strong opinions here)